hw34jdbctemplate/src/main/java/ua/ithillel/beans/Cart.java

```diff
@@ -0,0 +1,45 @@
+package ua.ithillel.beans;
+
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Cart {
+
+    private int cartId;
+    private int productId;
```

Comment:
in the task, it was necessary to implement the Cart class, which consists of a list of products added to this cart